### DISCUSSION TITLE
Welcome page

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#11044] [Feature Requests]
Maybe it is an idea or a question of how to do it?
My point is, whether it is possible to create a welcome page for a people who logs in to Directus. It can be a module with a system manual (not Directus manual) i.e. where and who has to enter and what. In can be in html or even better like a system WYSIWYG page to enter some welcome words...

**Comment by oreilles** [comment]
You could use a module for this. Modules are application blocks that appear in the application left sidebar. Here's the relevant documentation on how to create one: https://docs.directus.io/extensions/modules/
A module can export any Vue component, so you could model your "welcome page" exactly the way you want. 👍

**Reply by koksikus** [reply]
Thanks I will try to do it, although I have no experience ;) even in creating modules, maybe some advice ;). 
So I thought that it could be a nice option in Directus, when the previous user page does not exist it goes automatically to the welcome page...

**Reply by benhaynes** [reply]
Building a custom module is the first step, then we might want the ability to set that page as the default route globally or per role:

https://github.com/directus/directus/discussions/8751
https://github.com/directus/directus/discussions/4590

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
